vignettes/SEIRD_Model_Compartments.Rmd

```diff
@@ -0,0 +1,249 @@
+
+## The SEIaImIsRD Model
+The SEIaImIsRD model describes the following population groups:
+
+* **Susceptible** ($S$) - individuals that have not been exposed to the virus, which can be infected by infectious individuals;
+
+* **Exposed** ($E$) - individuals that have been exposed to the virus and have been infected, but are not yet infectious to others as they are within the incubation period;
+
+* Infectious - individuals that can spread the virus to others. In this model, we subdivide Infectious individuals into three sub-categories depending on their symptoms.  
+  * **Asymptomatic infectious** ($I_{asymptomatic}$, or $I_a$ for short);
+  * **Mildly symptomatic infectious** ($I_{mild}$, or $I_m$ for short),
+  * **Severely symptomatic infectious** ($I_{severe}$, or $I_s$ for short).
+Each category is set with different transmission parameters.
+
+* **Recovered** ($R$) - individuals that have recovered from the disease and obtained immunity to it. They cannot be infected again until they lose their immunity and return to the susceptible population.
+
+
+The SEIaImIsRD model consists of an ordinary differential equation (ODE) system of six ODEs, each describing the rate of the size changing of a population group evolves over time:
+
+
+$$\frac{\text{d}S}{\text{d}t} = -S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s) + \omega R,$$
+$$\frac{\text{d}E}{\text{d}t} = S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s) - \kappa E,$$
+$$\frac{dI_{a}}{dt} = \eta_{E->I_a} \kappa E - (\gamma_{I_a} + \mu_{I_a})I_{a},$$
+$$\frac{dI_{m}}{dt} = \eta_{E->I_m} \kappa E - (\gamma_{I_m} + \mu_{I_a})I_{m},$$
+$$\frac{dI_{s}}{dt} = \eta_{E->I_s} \kappa E - (\gamma_{I_s} + \mu_{I_a})I_{s},$$
+$$\frac{\text{d}R}{\text{d}t} =  \mu_{I_a}  I_{a} + \mu_{I_m}  I_{m} + \mu_{I_s}  I_{s} - \omega R,$$
+with the following parameters (values between 0 and 1 for all):
+* $\beta_{I_i}$ - the rate at which an infected individual of symptom group $i$ infects susceptible individuals ($i\in\{a, m, s\}$).
+* $\omega$ - the rate at which recovered individuals become susceptible again.
+* $\kappa$ - the rate at which exposed individuals become infectious. The incubation period thus lasts $1/\kappa$ days.
+* $\eta_{E->I_i}$ - the fraction of exposed individuals that, after the incubation period, became infectious of symptom group $i$. $\sum_i\eta_{E->I_i}=1$, $i\in\{a, m, s\}$.
+* $\gamma_{I_i}$ - the rate at which individuals of symptom group $i$ recover from the disease ($i\in\{a, m, s\}$).
+* $\mu_{I_i}$ - the rate at which individuals of symptom group $i$ die of the disease ($i\in\{a, m, s\}$).
+
+In addition to these six main ODEs above, the model also keeps track the number of 
+disease-related deaths ($D$) and the total number of cases ($C$) over time: $$\frac{\text{d}D}{\text{d}t} = \mu_{I_a} I_{a} +  \mu_{I_m}I_{m} +  \mu_{I_s} I_{s}$$, 
+$$\frac{\text{d}C}{\text{d}t} = S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s)$$.
```

Comment:
I think the full stop "." needs to be written within the dollar signs for the document to render properly.